### PR TITLE
initial commit for users management

### PR DESCRIPTION
**Pull request by PankajWorks**
- refactored config.go as per comments from Richard.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -80,6 +80,9 @@ generate-swagger: build-swagger-fix
 	rm -rf dataplane/api/client dataplane/api/model
 	swagger generate client -f http://$(CB_IP):$(CB_PORT)/cb/api/swagger.json -c client -m model -t dataplane/api
 	make fix-swagger
+generate-swagger-dp:
```
Comment (mhmxs) [inline Makefile]: Dockerized generator is missing

File: dataplane/config/config.go [hunk 2]
```diff
@@ -184,9 +216,15 @@ func WriteConfigToFile(baseDir, server, output, profile, workspace, token string
 	}
 
 	configList[profile] = Config{Server: server,
-		Workspace:    workspace,
 		Output:       output,
 		RefreshToken: token,
+		//Workspace: workspace,
```
Comment (mhmxs) [inline dataplane/config/config.go]: This line should be removed

File: dataplane/user/user.go [hunk 3]
```diff
@@ -2,41 +2,181 @@ package user
...
+		output.Write)
+}
+
+func revokeRolesImpl(client userClient, userid string, roleids []string, writer func([]string, utils.Row)) {
+	defer utils.TimeTrack(time.Now(), "assign roles to user")
+	log.Infof("[assignRolesImpl] sending assign roles request")
+	userRolesRequest := &model.RolesInput{
+		RoleIds: roleids}
+
+	//var deleteActionResponse *model.deleteActionResponse
```
Comment (mhmxs) [inline dataplane/user/user.go]: Please remove this line

File: dataplane/user/user.go [hunk 4]
```diff
@@ -2,41 +2,181 @@ package user
...
+	resp, err := client.DeleteRolesFromUser(users.NewDeleteRolesFromUserParams().WithUserID(userid).WithRoleIds(userRolesRequest))
+	if err != nil {
+		utils.LogErrorAndExit(err)
+	}
+	//deleteActionResponse = resp.Payload
```
Comment (mhmxs) [inline dataplane/user/user.go]: Please remove this line

File: Makefile (hunk 1 above)
Comment (PankajWorks) [reply Makefile]: I will create a defect for that. will good it as part of M3

Comment (PankajWorks) [reply Makefile]: https://hortonworks.jira.com/browse/BUG-115256

**Comment by PankajWorks** [comment]
This is not needed since the corresponding changes are merged

**Pull request closed without merging**